engineering_tools/opendrive_lanelet2_converter/opendrive2lanelet2convertor.py

```diff
@@ -115,6 +115,9 @@ def __init__(self, fn):
         self.relations = xml.Element('relations')
         self.all_nodes = []
         self.all_ways = []
+        self.projector_type = '1'
+        self.base_frame = '+proj=geocent +ellps=WGS84 +datum=WGS84 +units=m +no_defs'
+        self.target_frame = '+proj=tmerc +lat_0=38.95197911150576 +lon_0=-77.14835128349988 +k=1 +x_0=0 +y_0=0 +units=m +vunits=m'
```

Comment:
This target_frame should be read in from the xml file. I think you could just use self.geoReference as the target_frame.